addons/OPT/dynmarkerverwaltung/fn_markererstellenHover.sqf

```diff
@@ -0,0 +1,125 @@
+/**
+* Erstellten von Hover Markern im Overlay
+* 
+* Autor: [GNC]Lord-MDB
+*
+* Argumente:
+* 0: <STRING> _ID
+* 1: <STRING> _type
+* 2: <ARRAY> _farbe
+* 3: <STRING> _text
+* 4: <Number> _sichtbarkeit
+* 5: <ARRAY> _position
+* 5: <Number> _winkel
+* 6: <STRING> _textHover
+*
+* Rückgabewert:
+* 0: <STRING> _ID
+*
+* Server Only:
+* Nein
+* 
+* Lokal:
+* Ja
+* 
+* Global:
+* Nein
+* 
+* API:
+* Ja
+* 
+* Beispiel Externer Aufruf:
+* [] call EFUNC(dynmarkerverwaltung,markererstellenHover);
+* Beispiel interner Aufruf:
+* [] call FUNC(markererstellenHover);
+*
+*/
+
+#include "macros.hpp"
+
+
+params
+					
+[
+	["_id",""],
+	["_type",""],
+	["_farbe",[]],
+	["_text",""],
+	["_sichtbarkeit",1],
+	["_position",[]],
+	["_winkel",0],
+	["_textHover",""]
+];
+
+//Texturverweise
+
+//Spieler Marker
+#define player_icon "\A3\ui_f\data\map\vehicleicons\iconMan_ca.paa"
```

Comment:
Alle defines in eine eigene Datei auslagern, die in diesem Skript via ' #include'  eingefügt wird

Reply:
Außerdem ist es ein guter Stil, Konstanten komplett in Großbuchstaben zu schreiben. Das sollte hier auch gemacht werden. Dann sind die nämlich leichter von normalen Variablen zu unterscheiden